src/server/common.cc

```diff
@@ -190,6 +192,12 @@ bool ParseDouble(string_view src, double* value) {
   return true;
 }
 
+void OpArgs::RecordJournal(string_view key, ArgSlice args) const {
+  auto journal = shard->journal();
+  CHECK(journal);
+  journal->RecordEntry(txid, journal::Op::COMMAND, db_cntx.db_index, 1, make_pair(key, args));
```

Comment:
this records can be part of multi transaction and in this case we should use MULTI_COMMAND